host/bcm283x/dma.go

```diff
@@ -508,15 +516,8 @@ func (d *dmaChannel) isAvailable() bool {
 //
 // It doesn't clear the local controlBlock cached values.
 func (d *dmaChannel) reset() {
-	// Make sure nothing is happening.
 	d.cs = dmaReset
-	// Clear bits if needed.
-	d.cs = dmaEnd | dmaInterrupt
```

Comment:
Not sure. I removed those lines because they did not make the channel isAvailable(). Simple dmaReset did work. Anyway, the code is not enabled yet. I leave dmaInterrupt and will test later whether it is harmful.